### ISSUE TITLE
Add 3D puddles and water that reacts to a players body such as a splash in the puddle.

### ISSUE THREAD

**Issue by delebash** [GameDev4K/unreal-open-land#11]
I am new to unreal so please forgive me if I am asking bad questions.

Currently puddles are created using texture paint but it would be nice to be able to place puddles down that when I player walks through it the puddle splashes and maybe mud gets on the shoes.

One thing I am learning is that 2D texturing for visualization is used a lot, I guess for performance reasons..  The 2D texturing is not very realistic and I would prefer using 3D assets that respond like it would in the real world such as walking on sand and dynamically leaving footprints versus the method now of just painting a picture of sand with footprints.  Since UE5 can support the higher poly count it would be wonderful to move towards real life interaction with the landscape versus just texture painting.  Thanks for your hard work!

**Comment by arunoda** [comment]
@delebash yes interactive/dynamic effects in something I haven't touch in here. I don't have a roadmap for that yet. But, I will add support for that in the coming months.

For now, I think there are a good number of projects on the marketplace. (But, I haven't try any of that)
